src/getHumanAge.test.js

```diff
@@ -3,8 +3,36 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
+  it('should be declared', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  it(`should return an array`, () => {
+    expect(getHumanAge()).toBeInstanceOf(Array);
+  });
+
+  it(`should retern an array with 2 values`, () => {
+    expect(getHumanAge(9, 2)).toHaveLength(2);
+  });
+
+  it(`should retern [0, 0] if animal's 
+  age is less than 15 not inclusive`, () => {
+    expect(getHumanAge(10, 10)).toEqual([0, 0]);
+  });
+
+  it(`should retern 1 human year if animal's 
+  age is between 15 and 23 years inclusive`, () => {
+    expect(getHumanAge(15, 23)).toEqual([1, 1]);
+  });
+
+  it(`should add 1 human year for every next 4 cat's years
+   starting from 24 year inclusive`, () => {
```

Comment:
It's not a good description for this check. For 24-27 should return 2. And from 28 add 1 human year for every next 4 cat's years